src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/JSON1008.ValueTooSmall_LocationId_Valid.sarif

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "Sarif Functional Testing",
+          "informationUri": "https://www.example.com",
+          "version": "1.0"
+        }
+      },
+      "results": [
+        {
+          "ruleId": "TEST1001",
+          "message": {
+            "text": "Locations that specify file paths."
+          },
+          "locations": [
+            {
+              "id": 0,
```

Comment:
>0 [](http://example.com/codeflow?start=20&length=1)

add -1, the max allowed value

Reply:
will add the max
I think we should not add the -1;

-1 is the 3rd case in this test, 
"If id is absent, it SHALL default to -1"

and the spec does allow explicit -1:
 "id whose value is a non-negative integer"

[https://docs.oasis-open.org/sarif/sarif/v2.1.0/os/sarif-v2.1.0-os.html#_Toc34317672](https://docs.oasis-open.org/sarif/sarif/v2.1.0/os/sarif-v2.1.0-os.html#_Toc34317672)

Reply:
added the max